plugins/x86/x86_abi.ml

```diff
@@ -29,7 +29,16 @@ module SysV = struct
   let name = "sysv"
   let arch = `x86_64
   let stack n = Stack.create arch n
-  let arg = function
+
+
+  let xmm r =  Bil.(cast low 64 (var ymms.(r)))
+
+  let flt = function
+    | Ret_0 -> xmm 0
+    | Ret_1 -> xmm 1
+    | Arg n -> xmm n
```

Comment:
This function will blow up with an exception if we will try to pass more than 16 floats, and will be just incorrect if we will try to pass more than 8,  as according to SysV AMD64 ABI only xmms0-xmms7 are used for arguments passing, while the rest XMM8-XMM15 are used as scratch registers. So it should be something like this:
```ocaml
| Arg n -> if n < 8 then xmm n else stack Int.(n-8)
```

Note, this is still an under-approximation, as it will only work for monomorphic functions, but better than terminating analysis with an exception :)